docs/applications/development/clojure-deployment-with-immutant-and-wildfly-on-ubuntu-14-04.md

```diff
@@ -56,13 +56,13 @@ This guide will show how to deploy Clojure application to WildFly - widely used
 		sudo apt-get update
 		sudo apt-get install oracle-java8-installer
 
-	You will be asked twice to agree with "Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX" by installer. Press &lt;Ok&gt; button to agree the first time, and &lt;Yes&gt; button the second time.
+	You will be asked twice on screen to agree with "Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX." Press &lt;Ok&gt; button to agree the first time, and &lt;Yes&gt; button the second time.
```

Comment:
as per Formatting Guide "Use a Bold font weight for buttons, menu selections, important terms and words that require emphasis and anything that you want to stand out to the reader." - so looks like it is better to use *O* and *Yes* here